test/2.api/17.jobs.js

```diff
@@ -1,3 +1,61 @@
+var request = require("supertest");
+var should = require("should");
+var testUtils = require("../testUtils");
+request = request(testUtils.url);
+
+var API_KEY_ADMIN = "";
+var APP_ID = "";
+
+describe("Testing Jobs", function() {
+    it("Fetching jobs table data", function(done) {
+        request
```

Comment:
You have to have those lines in first test:

   API_KEY_ADMIN = testUtils.get("API_KEY_ADMIN");
   APP_ID = testUtils.get("APP_ID");
   APP_KEY = testUtils.get("APP_KEY");

Reply:
Like:
https://github.com/Countly/countly-server/blob/master/test/2.api/16.token.manager.js#L48-L52